src/tests/JIT/IL_Conformance/Convert/TestConvertFromIntegral.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestPriority>1</CLRTestPriority>
```

Comment:
@sandreenko I think you've already taken note of this, but I believe the reason CI has been green so far is because the outerloop has only been triggered for CoreCLR, and not Mono. Not sure what is the best course of action here.

Reply:
yes, thanks for reminding me. I have not found a pipeline that runs outerloop on mono.  Do we run them there nowadays?